.gitignore

```diff
@@ -23,5 +23,13 @@ tilt-settings.json
 tilt_config.json
 tilt.d/*
 
+# Common editor / temporary files
+*~
+*.tmp
+.DS_Store
+
+go.work
+go.work.sum
+
```

Comment:
Does this file needed for this PR?

Reply:
It's a minor ignore, unrelated to Go Bump but we don't think a separate PR is needed for this.